pkg/storaged/drives-panel.jsx

```diff
@@ -86,7 +86,7 @@ export class DrivesPanel extends React.Component {
                               name={name}
                               devname={block_name(block)}
                               detail={desc}
-                              go={() => cockpit.location.go([dev])}
+                              location={"#/" + dev}
```

Comment:
We should pass the original argument here: `location={[dev]}` and use `cockpit.location.go` in SidePanelRow below, to get the encoding right.  If we really need to use `href` of the Button, let's use `cockpit.location.encode` to encode the abstract location into a URL fragment.

Reply:
Thanks for taking a look. button links should use href for accessibility purposes  indeed - but here we haven't finalized yet if we should actually use links. But thanks for the pointer!